scenes/eternal_loom_sokoban/components/hud/sokoban_hud.tscn

```diff
@@ -57,3 +58,20 @@ layout_mode = 2
 theme = ExtResource("3_uhi48")
 theme_type_variation = &"HintLabel"
 text = "Reset"
+
+[node name="HBoxContainer3" type="HBoxContainer" parent="VBoxContainer"]
+layout_mode = 2
```

Comment:
IMO this should be above the other two, because otherwise the hud shifts upwards when the new action becomes visible:

[Screencast from 2025-05-22 14-05-46.webm](https://github.com/user-attachments/assets/c43b5996-74a7-42d5-be52-cf70f0548a38)